vendor/bat-native-ads/src/bat/ads/internal/ads/serving/choose/eligible_ads_predictor_util.h

```diff
@@ -160,6 +163,40 @@ CreativeAdPredictorMap<T> ComputePredictorFeaturesAndScores(
   return creative_ad_predictors_with_features;
 }
 
+template <typename T>
+std::vector<int> ComputeVoteRegistry(
+    const std::vector<T>& creative_ads,
+    const TextEmbeddingHtmlEventList& text_embedding_html_events) {
+  DCHECK(!creative_ads.empty());
+
+  std::vector<int> vote_registry(creative_ads.size(), 0);
+
+  for (const auto& text_embedding : text_embedding_html_events) {
```

Comment:
`text_embedding_html_event` instead of `text_embedding` as it is easier to read if we use the non-plural version of the list